EFCore.NamingConventions.Test/LowerCaseNamingTest.cs

```diff
@@ -14,15 +15,33 @@ public void Table_name_is_rewritten()
             Assert.Equal("simpleblog", entityType.GetTableName());
         }
 
+
```

Comment:
@adomorn please take care with newlines. Here there are many extra ones added, while below they're missing between the tests. Try to follow the existing style of the project.